### ISSUE TITLE
Investigate noise in Editor tests

### ISSUE THREAD

**Issue by bgrins** [WebKit/Speedometer#157]
We're looking into ways to make the editor tests less noisy in order to move them out of `tentative`. Areas to investigate:

* Squeeze more work into steps. For example - set the text to small, unhighlight, set the text back to big. Right now the actions are fairly limited because when adding an individual step for each the steps were very short.
* Make the input text bigger. In particular the "big" texts aren't actually that big. 
* Understand CodeMirror/TipTap highlighting, is it rAF based? If so, investigate rAF as setTimeout 0 like we did with the charting tests.
* @rniwa had mentioned that Chrome may be doing less layout work, and this issue can be a place to document any further details
* Adding a meta viewport tag (at least on the workload itself) to ensure that the same text gets rendered in each browser (#130)

**Comment by julienw** [comment]
Today I tried to look at the font-size difference between chrome and Firefox on Android, and it looks like this happens even with normal text. Chrome is making the text bigger, presumably to make it more readable, but I don't find any reference or pointer about where or why this is implemented like this.

Only when using `width=device-width` in the viewport meta tag there's no difference, but with the current `width=850` there is one. Currently we don't use `initial-scale=1`. If we use it we get a different behavior, but Chrome and Firefox are still different.

I pushed some examples in https://julienw.github.io/font-size-mobile/ and it would be good if some Google folks (ping @flashdesignory :D) could find out where this Chrome behavior comes from.

If we can't fix this cross-browser issue with our current `width=850` value, I think that our next step would be to use `width=device-width` and resize the iframe (for example with `max-width: 100%`?) so that on one specific device we have the same behavior in all browsers. But then the devices won't be comparable with each other (maybe that's already the case?).
